axis_backend/apps/clients/services/industry_service.py

```
@@ -0,0 +1,299 @@
+        root_items = []
+
+        for industry in industries:
+            tree_map[industry.id] = {
+                'id': industry.id,
+                'name': industry.name,
+                'code': industry.code,
+                'parent_id': industry.parent_id,
+                'children': []
+            }
+
+        for industry_id, industry_data in tree_map.items():
```

Ruff (0.14.5)

Loop control variable `industry_id` not used within loop body

Rename unused `industry_id` to `_industry_id`

(B007)